optimum/onnxruntime/utils.py

```diff
@@ -271,3 +271,23 @@ class ORTQuantizableOperator(Enum):
     Resize = "Resize"
     AveragePool = "AveragePool"
     Concat = "Concat"
+
+
+def _get_external_data_paths(src_file_names, dst_file_names):
+    import onnx
```

Comment:
I can see the reason to put imports here if it is only used in this method - but is this standard? Actually never sure about it